### PR TITLE
Update status indicator colors.

### PR DESCRIPTION
**Pull request by ImogenF**
### Summary

<blockquote class="trello-card"><a href="https://trello.com/c/ZDktty09/264-status-indicator-tweak-grey-add-yellow-in-progress">Status indicator: tweak grey, add yellow (in progress)</a></blockquote>

### To be tested

Tester 1 (Peter)

- [x] No error/warning in the console
- [x] The netlify build is working

Tester 2

- [ ] No error/warning in the console
- [ ] The netlify build is working

### CODE REVIEW

**Comment by ImogenF** [comment]
[Status indicator: tweak grey, add yellow (in progress)](https://trello.com/c/ZDktty09/264-status-indicator-tweak-grey-add-yellow-in-progress)

File: packages/components/src/Status/Status.tsx [hunk 1]
```diff
@@ -18,7 +23,7 @@ const getColorFromProps = ({ success, error, theme }: Props): string => {
     return theme.color.error
   }
 
-  return theme.color.background.dark
+  return "#989898"
```
Comment (TejasQ) [inline packages/components/src/Status/Status.tsx]: Why can't we use a constant here?

File: packages/components/src/utils/constants.ts [hunk 2]
```diff
@@ -74,6 +74,7 @@ const borderColors = {
 const color = {
   primary: primaryColor,
   disabled: "#f5f5f5",
+  running: "#ffbd41",
```
Comment (TejasQ) [inline packages/components/src/utils/constants.ts]: Can we call this `warn` instead? I foresee it being used for that case more than to indicate a _running_ state.

File: packages/components/src/Status/Status.tsx (hunk 1 above)
Comment (ImogenF) [reply packages/components/src/Status/Status.tsx]: We could. My thinking was that there are so many greys in the constants used for so many things that it doesn't really make sense to keep adding to them every time we want a very particular grey. The closest constant we currently have is `theme.color.text.lightest`, as far as I can see, which doesn't really have anything to do with a neutral status. 

Should we maybe refactor the constants and have a list of greys, which `theme.color.text` etc can then make use of?

Comment (TejasQ) [reply packages/components/src/Status/Status.tsx]: @micha-f would it make sense to use one of the many _already existing_ greys for this case?

Comment (micha-f) [reply packages/components/src/Status/Status.tsx]: I think it'll be difficult to use a common set of greys for text, backgrounds, separators and shades. I'd say it's best to just use the specified grey here. I think design should go first. And this value looks good for a neutral status indicator.

File: packages/components/src/utils/constants.ts [hunk 3]
```diff
@@ -74,6 +74,7 @@ const borderColors = {
 const color = {
   primary: primaryColor,
   disabled: "#f5f5f5",
+  warn: "#ffbd41",
```
Comment (peterszerzo) [inline packages/components/src/utils/constants.ts]: Can we call this `warning`?

**Pull request merged**